stats.pl

```diff
@@ -0,0 +1,111 @@
+#!/usr/local/bin/perl
+
+#
+# Authentic Theme (https://github.com/authentic-theme/authentic-theme)
+# Copyright Ilia Rostovtsev <ilia@virtualmin.com>
+# Licensed under MIT (https://github.com/authentic-theme/authentic-theme/blob/master/LICENSE)
+#
+use lib ("$ENV{'PERLLIB'}/vendor_perl");
+use Net::WebSocket::Server;
+use utf8;
+use JSON;
+use threads;
```

Comment:
One last question ... how widely available is this `threads` module in Perl?

Reply:
It's always been a part of the standard Perl installation. It still is on Debian and Ubuntu, but the latest versions of Rocky and Alma started to make it an additional module. They first did it to `lib` and `open`, and then to many others.

So, I had to add it as a recommended package to Webmin RPM and DEB builds.

However, I'm now thinking that maybe we should just include it in our `vendor_perl` modules?

Reply:
Can we do that? It seems like a module that would require native code ...

Reply:
> Can we do that? It seems like a module that would require native code

No, we can't do it reliably because there are shared libraries involved.

We'll make sure it's installed. It's available on all systems I've tested, including FreeBSD. If not, then we just won't allow this feature.

Reply:
Ok makes sense!